feasibility-triangle/troubleshooting/test-cql.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
```

Comment:
there should be a check and if blazectl and jq are available and if false an appropriate message should be displayed explaining to the user where to get it.
At the moment if one does not have blazectl it fails silently

Reply:
ok added checks for both